hugo/layouts/partials/projects/blog.html

```diff
@@ -0,0 +1,13 @@
+<section class="blog band band_grey" id="blog-container">
```

Comment:
Nope. This has the h1 in another place. Would need to be configured via more hugo `dicts` and rather die